README.md

```diff
@@ -90,7 +90,7 @@ For more models, go to [**tensorflow/swift-models**][swift-models].
 ### Requirements
 
 * [Swift for TensorFlow toolchain][toolchain].
-* An environment that can run the Swift for TensorFlow toolchains: Linux 18.04 or macOS with Xcode 10.
+* An environment that can run the Swift for TensorFlow toolchains: Ubuntu 18.04, macOS with Xcode 10 or Windows 10.
```

Comment:
Oxford comma please:
```suggestion
* An environment that can run the Swift for TensorFlow toolchains: Ubuntu 18.04, macOS with Xcode 10, or Windows 10.
```

Otherwise, the sentence end can be misinterpreted as "macOS with (Xcode 10 or Windows 10)".